### DISCUSSION TITLE
Does the data cache grow forever?

### DISCUSSION THREAD

**Discussion by christopher-drifte** [vercel/next.js#67476] [App Router]
When fetch requests happen, they are cached in `.next/cache`. Revalidate causes old files to be rewritten, but stale files do not appear to be removed (or at least, they weren't during my experimentation). Does NextJS automatically clean up old cache files, and - if so - is there a way to limit the size of the data cache?

**Comment by Arnoldnuo** [comment]
wating for an answer

**Reply by ratz6** [reply]
any update @nextjs-bot  ?
